bridge-worker/bridge_worker.go

```diff
@@ -185,8 +202,19 @@ func RunBridgeWorker(
 		}(job)
 	}
 
+	go func() {
+		defer cancelCtx()
+		env.server.Start()
+		env.logger.Warn("Http server stopped")
```

Comment:
Nit: Let's use lowercase for log messages:
```suggestion
		env.logger.Warn("http server stopped")
```